Nil pointer dereference in config_fragment is Created not set

Created is not a required field. However, unless we set it in the fragment, the program crashes with the following error:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x8590b7]

goroutine 1 [running]:
github.com/tweag/rules_img/src/cmd/manifest.overlayConfigFromFile(0xc0001f15b8, {0x7ffff6172df0?, 0xc0001f15d0?}, 0x0)
        external/rules_img_tool+/cmd/manifest/manifest.go:343 +0xb37
github.com/tweag/rules_img/src/cmd/manifest.prepareConfig({_, _, _})
        external/rules_img_tool+/cmd/manifest/manifest.go:212 +0x19f
github.com/tweag/rules_img/src/cmd/manifest.ManifestProcess({0xd5b2d0?, 0x12ce880?}, {0xc00011e028, 0x24, 0x25})
        external/rules_img_tool+/cmd/manifest/manifest.go:96 +0xa85
main.Run({0xd5b2d0, 0x12ce880}, {0xc00011e008, 0x26, 0x27})
        external/rules_img_tool+/cmd/img/img.go:62 +0x273
main.main()
        external/rules_img_tool+/cmd/img/img.go:122 +0x36
```

--- a/src/cmd/manifest/manifest.go
+++ b/src/cmd/manifest/manifest.go
@@ -340,7 +340,7 @@ func overlayConfigFromFile(config *specv1.Image, filePath string, isBase bool) e
 
 	// inherit some fields if this is not a base config
 	if !isBase {
-		if !configFragment.Created.IsZero() {
+		if !(config.Created == nil) && !configFragment.Created.IsZero() {
 			config.Created = configFragment.Created
 		}
 		if configFragment.Author != "" {

